resources/views/device-group/index.blade.php

```diff
@@ -57,6 +57,39 @@ class="fa fa-trash" aria-hidden="true"></i></button>
             </div>
         </div>
     </div>
+    <div class="container-fluid">
+        <div id="manage-device-groups-panel" class="panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">
+                @lang('ungrouped Devices') ({{ $ungrouped_devices->count() }})
```

Comment:
Should be title case `Ungrouped Devices`

Reply:
@murrant 
fixed title case :-)